src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/PoisonUsage.txt

```diff
@@ -1,1328 +1,1328 @@
 <PrebuiltLeakReport>
-  <File Path="dotnet-sdk-x.y.z/sdk/x.y.z/Roslyn/bincore/System.Reflection.Metadata.dll">
-    <Type>AssemblyAttribute</Type>
-  </File>
-  <File Path="dotnet-sdk-x.y.z/sdk/x.y.z/Roslyn/bincore/System.Text.Encoding.CodePages.dll">
+  <File Path="dotnet-sdk-x.y.z/sdk/x.y.z/Roslyn/bincore/runtimes/win/lib/netcoreapp3.1/System.Text.Encoding.CodePages.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="dotnet-sdk-x.y.z/sdk/x.y.z/Roslyn/bincore/System.Runtime.CompilerServices.Unsafe.dll">
+  <File Path="dotnet-sdk-x.y.z/sdk/x.y.z/Roslyn/bincore/runtimes/win/lib/netcoreapp3.1/System.Text.Encoding.CodePages.dll">
```

Comment:
It appears that there are duplicates.  Are duplicates because of version differences or because their source is different?

Reply:
I think this is related to why we're not seeing Hash failures as well as AssemblyAttribute ones - I think something may be short-circuiting to the first failure cause rather than adding the additional cause.  As such I'm inclined to say that we shouldn't block on it unless you disagree.  I think it's fine to leave the duplicates in for now while I'm working on the fix for missing failure types.

Reply:
Agreed